x-pack/plugins/security_solution/public/index.ts

```diff
@@ -11,3 +11,5 @@ import { PluginSetup, PluginStart } from './types';
 export const plugin = (context: PluginInitializerContext): Plugin => new Plugin(context);
 
 export { Plugin, PluginSetup, PluginStart };
+
+export * from './resolver';
```

Comment:
The test plugin needs to use resolver directly. In order to do that, we need to export it (and various helpers) from the security solution plugin.